Jenkinsfile

```diff
@@ -0,0 +1,17 @@
+pipeline {
+  agent {
+      docker { image 'jamesl22/lit-ci' }
+  }
+  stages {
+    stage('Test lit') {
+      steps {
+        sh 'rm -rf lit ../lit'
+        sh 'git clone https://github.com/Varunram/lit'
+        sh 'mv lit ../'
+        sh 'cd ..'
+        sh 'make lit lit-af'
+        sh 'make test with-python=true'
+      }
```

Comment:
Imho, this should be broken up into multiple pipeline stages.  And there should be a way to start with a "clean" workspace pointing at the commit that's being built at the beginning of the script. ([link](https://zwischenzugs.com/2017/04/23/things-i-wish-i-knew-before-using-jenkins-pipelines/)]

Reply:
well, jenkinsfile is really weird, I guess I don't want to mess around too much with it - I wanted it to work with minimal stuff, hence I didn't. But yes, can be expanded upon.